src/amqp/amqpsender.h

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2017 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef ZCASH_AMQP_AMQPSENDER_H
+#define ZCASH_AMQP_AMQPSENDER_H
+
+#include "amqpconfig.h"
+
+#include <deque>
+#include <memory>
+#include <future>
+#include <iostream>
+
+class amqp_sender : public proton::messaging_handler {
```

Comment:
Class names are usually UpperCamelCase. (I realise Proton doesn't follow this convention for its classes, but I think we should be consistent with other Zcash code here.)